CMakeLists.txt

```diff
@@ -1,17 +1,27 @@
 # First-order language CUDA requires at least CMake 3.18
 cmake_minimum_required(VERSION 3.24)
 
+# Default seletion of CUDA Compute Capabilities.
+# This must be called before project() or cmake sets it to the oldest non-deprecated CC
+# "all" and "all-major" work for Intel and perhaps for ARM with discrete GPUs, but not Tegra and Jetson.
+if(EXISTS "/etc/nv_tegra_release")
```

Comment:
Thanks!
Tegras are built around specialized ARM cores that are combined with NVidia cores in the same chip. People use it with several flavours of Linux, we have Ubuntu. But since there is now Windows on ARM, who knows, there may be Windows on Tegra in the future.